notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb

```diff
@@ -0,0 +1,1424 @@
+{
```

Comment:
<p>This line should be in markdown, right now it's rending as code so throws as error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Good catch! Done!</p>